src/AzureAuth/CommandMain.cs

```diff
@@ -397,18 +442,20 @@ private int GetToken()
                     try
                     {
                         results = authFlowExecutor.GetTokenAsync().Result.ToArray();

```

Comment:
Why are we using array instead of the returned list?

Reply:
We return IEnumerable<T>. So we can either convert the results to List or Array. Array is chosen here since we do not need to manipulate (add/update/remove) the results as we only need to access the contents of it.